### PR TITLE
Improve properties display

### PR DESCRIPTION
**Pull request by by-robots**
Changes the component properties display to a system component and then improves the amount of detail given.

Screenshot 2019-08-07 at 15 50 50

### CODE REVIEW

File: lib/fenrir_view/property_types.rb [hunk 1]
```diff
@@ -74,9 +74,11 @@ def component_property_rule_descriptions
             ('Required' if data[:required]),
             ("As #{data[:one_of_type].join(' or ')}" if data[:one_of_type]),
           ].compact.join('. '),
+          default: data[:default],
```
Comment (jsundt) [inline lib/fenrir_view/property_types.rb]: Let's remove the info key above here if it's no longer in use.

File: lib/fenrir_view/components/fenrir_view_properties/fenrir_view_properties.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  examples:
+  - name: "Basic Property List"
+    properties:
+      property_list:
+        Ruby:
+          required: true
+          one_of_type:
+            - 'Bool'
+          default: true
+        JavaScript:
```
Comment (jsundt) [inline lib/fenrir_view/components/fenrir_view_properties/fenrir_view_properties.yml]: Could we add an additional test case for properties that has a symbol value?

I believe the current implementation of the properties table renders symbols as strings which is confusing as `symbol` and `:symbol` won't be the same thing...

File: lib/fenrir_view/components/fenrir_view_properties/fenrir_view_properties_facade.rb [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    # Returns a text representation of a boolean value or a stringified version
+    # of a data structure.
+    if !!property[:default] == property[:default]
+      return property[:default] ? 'true' : 'false'
+    elsif property[:default].is_a?(Array) || property[:default].is_a?(Hash)
+      return property[:default].inspect
+    end
+
+    # Cast everything else as a string.
+    property[:default].to_s
```
Comment (jsundt) [inline lib/fenrir_view/components/fenrir_view_properties/fenrir_view_properties_facade.rb]: `is_a?(Symbol)` should do something like `return ":#{property[:default]}"`

**Comment by jsundt** [comment]
When you feel like it's ready squash these commits into 1 and add a second commit that changes news.md and the version file :)

**Comment by by-robots** [comment]
> When you feel like it's ready squash these commits into 1 and add a second commit that changes news.md and the version file :)

All done! 🎉

**Pull request merged**